### PR TITLE
sitemap-generator: Fixed event callback type

### PR DESCRIPTION
**Pull request by meekay**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - Documentation: https://github.com/lgraubner/sitemap-generator/tree/master#events
  - Source code: https://github.com/lgraubner/sitemap-generator/blob/master/src/index.js

### CODE REVIEW

**Comment by meekay** [comment]
Hello @grgr-dkrk, could you please [review this PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66290) for `@types/sitemap-generator`, as it needs your approval before it can be merged. Thank you!

File: types/sitemap-generator/sitemap-generator-tests.ts
Comment (peterblazejewicz) [inline types/sitemap-generator/sitemap-generator-tests.ts:27]:
```suggestion
});
generator.on('add', () => {});
```
add this, to verify, it's backward compatible

File: types/sitemap-generator/sitemap-generator-tests.ts
Comment (peterblazejewicz) [inline types/sitemap-generator/sitemap-generator-tests.ts:31]:
```suggestion
});
generator.on('ignore', () => {});
```
same, it verifies change is backward compatible

**Comment by peterblazejewicz** [comment]
@meekay please retain those changed lines as in comments, I'll aprove the change, thx!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by meekay** [comment]
Ready to merge

**Pull request merged**
